libs/community/langchain_community/llms/cloudflare_workersai.py

```diff
@@ -29,22 +29,27 @@ class CloudflareWorkersAI(LLM):
             cf_ai = CloudflareWorkersAI(
                 account_id=my_account_id,
                 api_token=my_api_token,
+                ai_gateway=my_cfai_gateway_token,
                 model=llm_model
             )
     """  # noqa: E501
 
     account_id: str
     api_token: str
+    ai_gateway: str = ""
```

Comment:
Let's put these new kwargs at the end in case users are passing in positionally.